library/lib_lazcalc.ks

```diff
@@ -1,80 +1,56 @@
 //=====LAUNCH AZIMUTH CALCULATOR=====
 //~~LIB_LAZcalc.ks~~
-//~~Version 1.0~~
+//~~Version 1.1~~
```

Comment:
Given that pretty much the entire structure of the function has changed, I'd be willing to call this v2.0